### PR TITLE
translation

### PR DESCRIPTION
**Pull request by fabianfrz**
cleanup in
* interfaces_bridge_edit.php 
* services_upnp.php
* services_dhcpv6.php
* interfaces_lagg_edit.php

### CODE REVIEW

File: src/www/services_dhcpv6.php [hunk 1]
```diff
@@ -608,7 +608,7 @@ function show_netboot_config() {
                           <?=gettext("Change DHCPv6 display lease time from UTC to local time."); ?>
                         </strong>
                         <div class="hidden" for="help_for_dhcpv6leaseinlocaltime">
-                          <strong><?=gettext("Note:");?></strong> <?=gettext("By default DHCPv6 leases are displayed in UTC time.  By checking this box DHCPv6 lease time will be displayed in local time and set to time zone selected.  This will be used for all DHCPv6 interfaces lease time."); ?>
+                          <?=gettext("By default DHCPv6 leases are displayed in UTC time. By checking this box DHCPv6 lease time will be displayed in local time and set to time zone selected. This will be used for all DHCPv6 interfaces lease time."); ?>
```
Comment (fichtner) [inline src/www/services_dhcpv6.php]: Thanks for this :)

**Comment by fichtner** [comment]
This will be the last merge of translation cleanups before the 16.7 release. We don't want to risk introducing more problems and lowering the number of provided translations now. Let me know when this can be merged.

(Templates will be regenerated once more before 16.7 is out.)

**Comment by fabianfrz** [comment]
@fichtner you can merge it - I can wait for the new template (my tool does not find many which are left)

**Comment by fichtner** [comment]
Merged, thanks for all your work! :)

**Pull request merged**